scripts/cloud-hooks/hooks/common/post-code-deploy/post-code-deploy.sh

```diff
@@ -20,9 +20,13 @@ repo_url="$5"
 repo_type="$6"
 
 
+acsf_file="/mnt/files/$AH_SITE_GROUP.$AH_SITE_ENVIRONMENT/files-private/sites.json"
```

Comment:
It would be better to use the provided script variables here (i.e. `$site.$target_env`), rather than environment variables, so that this script can be tested more easily.